app/services/strava_service.rb

```diff
@@ -0,0 +1,19 @@
+class StravaService
+  attr_reader :connection
+
+  def initialize(key)
+    header_key = "Bearer #{key}"
+    @connection = Faraday.new(url: "https://www.strava.com/api/v3")
+    @connection.headers = {'Authorization' => header_key}
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.